controller.py

```diff
@@ -832,7 +939,7 @@ def presentation_ready_work_for(self, identifier):
             return work
         return None
 
-    def process_urns(self, urns, collection_details=None, **kwargs):
+    def process_urns(self, urns, metadata_identifier=None, **kwargs):
```

Comment:
I changed `collection_details` to `metadata_identifier` throughout to make it more specific. I had to also make sure to change this in `routes.py`, where we pass keyword arguments into methods like this one, and our unit tests don't have a way of checking whether the arguments match -- only that the controller has a method with the right name.